mfr/ext/tabular/render.py

```diff
@@ -50,12 +54,18 @@ def render_html(fp, src=None):
     table_size = 'small_table' if len(columns) < 9 else 'big_table'
     slick_grid_options = config.get('slick_grid_options').get(table_size)
 
+    try:
+        columns = json.dumps(columns)
+        rows = json.dumps(rows)
+    except UnicodeDecodeError:
+        raise UnexpectedFormattingException()
```

Comment:
The default error message that will be displayed is `Unable to render. Download(link) file to view it. [any additional msg goes here]`